services/skipchain/skipchain.go

```diff
@@ -456,6 +461,20 @@ func (s *Service) lenSkipBlocks() int {
 	return len(s.SkipBlocks)
 }
 
+// saves the actual identity
+func (s *Service) save() {
```

Comment:
Something like (pseudo code) ...?
```
func (c *Context) Save(file string, i interface{}) error {
   data := network.MarshalRegisteredType(i)
   return fileIO.Write(data)
}
```
You're right, there are already two uses case so it may be time :)